docs/en/changes/changes.md

```diff
@@ -78,6 +78,7 @@
 * Fix TCP service instances are lack of instance properties like `pod` and `namespace`, which causes Pod log not to work for TCP workloads.
 * Add Python HBase happybase module component ID(94).
 * Fix gRPC alarm cannot update settings from dynamic configuration source.
+* [**Breaking Change**] Optimize single trace query performance by customizing routing in ElasticSearch
```

Comment:
Correct me if I'm wrong, although you declare this is a breaking change, users can still use the storage from previous version (without any warnings / errors), but they won't be able to search old traces by trace id because the routing field is absent when indexing but the routing field is present when querying from UI, is that the case?

Reply:
yes

Reply:
@wu-sheng read this thread, I believe this would cause user confusions when they upgrade from an older release.

Reply:
Yes, I had read so. What do you suggest? Do you suggest we intend to breaking something or somehow raise a warning?
I believe this change is forward positive improvement.